AVIenabledbyArc/vi-edge-deployment-script.sh

```diff
@@ -52,24 +57,51 @@ function create_cognitive_hobo_resources {
   result=$(az rest --method post --uri $createResourceUri 2>&1 >/dev/null || true)
   echo $result    
 
-  if [[ "$result" == *"ERROR:"* ]]; then
-    echo "CS Resources already exist. ignoring"
+  if [[ "$result" == *"ERROR: Conflict"* ]]; then
+    echo "CS Resources already exist. Moving on."
   else
-    echo "sleeping for $sleepDuration seconds"  
+    echo "No CS resources found. Creating"  
+  fi
+  getSecretsUri="https://management.azure.com/subscriptions/${viSubscriptionId}/resourceGroups/${viResourceGroup}/providers/Microsoft.VideoIndexer/accounts/${viAccountName}/ListExtensionDependenciesData?api-version=${viApiVersion}"
+  csResourcesData=$(az rest --method post --uri $getSecretsUri 2>&1 >/dev/null || true)
```

Comment:
Line 66 replicate line 73.

You may want to do:

do{

} while (condition) 

this way, you can avoid it

Reply:
Done !